mParticle-Apple-SDK/Identity/MPIdentityApi.m

```diff
@@ -469,8 +470,7 @@ - (BOOL)aliasUsers:(MPAliasRequest *)aliasRequest {
     }
     
     dispatch_async([MParticle messageQueue], ^{
-        
-        [MParticle.sharedInstance.backendController skipNextUpload];
+        [(MParticleKit *)[MParticle.sharedInstance kitInstance:[MParticleKit kitCode]] skipNextUpload];
```

Comment:
What is the purpose of `skipNextUpload`? We don't have a similar concept in Web at the moment, and if this is new, what is the intention here?